docs/sources/googlecloudauditlogs.md

```diff
@@ -0,0 +1,63 @@
+# Event Source for Google Cloud Audit Logs
+
+This event source receives messages from a [Google Cloud Audit Logs][gc-auditlogs] Sink by subscribing
+to a [Google Cloud Pub/Sub][gc-auditlogs-events] topic.
+
+## Prerequisite(s)
+
+- Service Name: The name of the API service performing the operation. For example, "pubsub.googleapis.com".
+- Method Name: The name of the service method or operation. For API calls, this should be 
+               the name of the API method. For example, "google.pubsub.v1.Publisher.CreateTopic".
+- Resource Name (Optional): The resource or collection that is the target of the operation. The name is
+                            a scheme-less URI, not including the API service name. [Google Cloud Audit Logs Types][gc-auditlogs-types]
```

Comment:
It feels to me like these should be somewhere in the "Deploying an Instance of the Source" section, since they describe what goes in the web form and not something users have to create.

I let you decide what you think is best.

Reply:
Makes sense, I've moved to "Deploying an Instance of the Source"